script.transmission/addon.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.transmission"
+       name="Transmission"
+       version="0.8.0"
+       provider-name="Correl Roush / Vasyl Gello">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0" />
+    <import addon="script.module.beautifulsoup4" version="4.8.2" />
+    <import addon="script.module.kodi-six" version="0.1.3"/>
+    <import addon="script.module.requests" version="2.22.0"/>
+    <import addon="script.module.simplejson" version="3.17.0"/>
```

Comment:
not needed anymore

Reply:
Removing these imports complaints on missing kodi_six. Or do you mean kodi-six should not be used here?

Reply:
I mean any reference to simplejson, this was only for helix and below